.github/CONTRIBUTING.md

```diff
@@ -13,46 +15,45 @@ Also, please be sure to follow our [code of conduct](https://www.classicpress.ne
+  1. The change impacts existing ClassicPress users. Otherwise, there are literally thousands of things we could look at, but we need to prioritize our development time. Right now the best tool we have for this is [petitions](https://forums.classicpress.net/c/governance/petitions/77).
+  2. The change is not going to break backward compatibility, and has minimal effects on the existing plugin and theme ecosystem. A good way to evaluate the effects of a change on plugins or themes is to do a search on [wpdirectory](https://wpdirectory.net). (Major changes are also a possibility but require a planning effort around when and how they will be released, as well as agreement from the community per our [democratic process](https://www.classicpress.net/democracy/).)
   3. The change has automated tests.
   4. We understand the code change very well or can ask questions of someone who understands it very well.
 
 If your change meets all of these criteria then we will most likely accept it.
 
-We take a lot of care with our platform, and ClassicPress is maintained by volunteers in our free time. This means there may be some delays in reviewing and merging pull requests.
-
 ## What to work on?
 
 If you're not sure where to start contributing, here are some ideas:
 
-  - Review and test [existing PRs](https://github.com/ClassicPress/ClassicPress/pulls), especially looking at how well they fit the criteria described above. Let us know how you tested the changes and what you found. We need to be thorough and careful with any changes we make, and the more eyes we can get on our PRs the better.
+  - Review and test [existing PRs](https://github.com/ClassicPress/ClassicPress/pulls), especially looking at how well they fit the criteria described above. Let us know how you tested the changes and what you found. Screenshots, gifs, video and text instructions documenting the tests can be very useful. We need to be thorough and careful with any changes we make, and the more eyes we can get on our PRs the better.
   - Bugfixes, you can find a list of known bugs on our [issues page](https://github.com/ClassicPress/ClassicPress/issues).
   - Issues with the [`help wanted`](https://github.com/ClassicPress/ClassicPress/labels/help%20wanted) or [`good first issue`](https://github.com/ClassicPress/ClassicPress/labels/good%20first%20issue) labels.
-  - [Planned](https://petitions.classicpress.net/?view=planned) or [most wanted](https://petitions.classicpress.net/?view=most-wanted) feature requests from our petitions site.
-  - Exploratory PRs with your own suggested changes, but please remember these will be subject to review to make sure they are in line with the project's direction.
+  - [Planned Milestones](https://github.com/ClassicPress/ClassicPress/milestones) or exploratory PRs with your own suggested changes. Please remember these will be subject to review to make sure they are in line with the project's direction.
```

Comment:
I think adding milestones is good but it should probably go with the sentence above instead (the one that was removed).

Reply:
I guess we can do that for now until we retire the petitions subdomain. However, it now says, "it is archived." All the petitions are on the forums. We are using "hearts" to poll until a properway to do this is chosen. Would you still suggest we keep pointing to this?

Reply:
OK, I see why that was removed. The archive domain will stay up (to keep old links working, point people to the new location, and show a better view of what people decided they wanted to see in ClassicPress), but it will become more outdated over time. Also, jumping right in with those petitions is probably not the best place to get started with contributing.

Reply:
I think that resolves it.